flake.nix

```diff
@@ -162,7 +162,7 @@
               description = "Spell checking";
               entry = "typos";
               pass_filenames = true;
-              excludes = [ "contract-bindings/artifacts" ];
+              excludes = [ "contract-bindings" ];
```

Comment:
The excludes should all go into `.typos.toml` and I think can be deleted here.

Reply:
thanks for identifying this dupe, updated https://github.com/EspressoSystems/espresso-sequencer/commit/241141ba5e41883171da720b3e4741cb0aeec3b4